docs/program/apis/sessions.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Session Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the session management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+Normally, sessions are managed for you with default resource configuration, but if you want to manage sessions yourself, you can with Viam's sessions management API.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server`.
+The period of time during which these clients are present is called a *session*.
+Imagine a session as analogous to the lifetime of a robot, in regards to its network connection.
```

Comment:
```suggestion
```
I think this is confusing and doesn't add, because when I think of a "lifetime of a robot" I think of its entire lifetime, not just a session.

Reply:
You might imagine the lifetime of a robot differently, and that makes sense, but I still find it helpful to imagine a session in that way! is it ok if I leave this up or do you find it completely nonsensical? I think this whole thing will be a lot easier for users to understand w the clients ecosystem diagram thing